tekton/cronjobs/robocat-cadmin-cron/cronjob.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: folder-cd-trigger
+spec:
+  schedule: "0 3 * * *"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: trigger
+            env:
+              - name: GIT_REPOSITORY
+                value: "github.com/afrittoli/plumbing"
+              - name: GIT_REVISION
+                value: "issues/224"
```

Comment:
are these values temporary? i guess you want to update them after the merge?

(we need to get some kinda way setup to apply changes that are within a pull request more smoothly one day! :D)

Reply:
upz, yeah, they were not meant to be added in the PR - I was just trying to test the setup in the dogfooding repo.

Reply:
> (we need to get some kinda way setup to apply changes that are within a pull request more smoothly one day! :D):

+100 - that's the direction we want to go for sure!